src/playlist-controller.js

```diff
@@ -1223,6 +1251,26 @@ export class PlaylistController extends videojs.EventTarget {
       Number.MAX_VALUE : duration;
   }
 
+  /**
+   * Handle the startstreaming event on the ManagedMediaSource.
+   * This event indicates that segment loading should be started/resumed.
+   *
+   * @private
+   */
+  handleStartStreaming_() {
+    this.load();
+  }
+
+  /**
+   * Handle the endstreaming event on the ManagedMediaSource.
+   * This event indicates that segment loading should be paused.
+   *
+   * @private
+   */
+  handleEndStreaming_() {
+    this.pause();
+  }
+
```

Comment:
Do we really need these wrappers for load and pause?

Reply:
We don't. If the logic ever becomes more sophisticated that just load/pause in the future, then we would need them. We can cross that bridge if we get there, I'm fine removing them now

Reply:
yeah, at this point seems redundant